src/base.ts

```diff
@@ -69,6 +69,26 @@ export interface IGetProjectsResponse {
   projects: IProject[];
 }
 
+export interface IDomains {
+  _id: string;
+  name: string;
+  type: string;
+  project: {
+    _id: string;
+    project_id: string;
+  };
+  status: string;
+  certificatesStatus: string;
+  redirectTo: '';
```

Comment:
Why is this an empty string?

Reply:
I think this shows that this domain is going to be redirected to, that's why I saw it, because it's not needed, I left it blank.

Reply:
I fixed it